lago/cmd.py

```diff
@@ -899,7 +900,18 @@ def create_parser(cli_plugins, out_plugins):
     return parser
 
 
+def signal_handler(signum, frame):
+
```

Comment:
"signal_handler" is a too generic name, won't it be better to call it "sigterm_handler", or better, just give it a name that tells what it does, so the link between the signal and the activity it triggers becomes obvious in the 'signal.signal' line.